tests/convert_latin1_to_utf8_tests.cpp

```diff
@@ -0,0 +1,37 @@
+#include "simdutf.h"
+
+#include <array>
+#include <iostream>
+
+#include <tests/helpers/transcode_test_base.h>
+#include <tests/helpers/random_int.h>
+#include <tests/helpers/test.h>
+#include <memory>
+
+namespace {
+  using simdutf::tests::helpers::transcode_utf8_to_utf16_test_base;
+
+  constexpr size_t trials = 10000;
```

Comment:
I think that you want to use this variable, and add a loop in the test below (so that it tries 10000 times, with different randomly generated values).